### PR TITLE
fix: remove action and channel fields from Operation Trait Object

### PR DESCRIPTION
**Pull request by magicmatatjahu**
---
title: "remove action and channel fields from Operation Trait Object"
---

---

**Related issue(s):**
https://github.com/asyncapi/spec/issues/856

---

This PR only shows what changes should be done described in the mentioned issue (https://github.com/asyncapi/spec/issues/856). If we decide to leave these fields, I will close the PR.

I think the ability to define `channel` and `action` at the trait level is bad, because both fields are required by the operation itself.

### CODE REVIEW

**Review by smoya — APPROVED** [review]

**Review by dalelane — APPROVED** [review]
makes sense to me

**Review by fmvilas — APPROVED** [review]
LGTM 👍

**Comment by derberg** [comment]
@magicmatatjahu fyi you have 3 approvals out of 5 codeowners, you can merge when you're ready

will you update JSON Schema?

**Comment by derberg** [comment]
> If 3 people are in favor of it

in favour of what 😅

**Comment by magicmatatjahu** [comment]
Of my changes in this PR 😆

**Comment by smoya** [comment]
> Of my changes in this PR 😆 

You have 3 approvals

**Pull request merged**
